ktor-http/common/src/io/ktor/http/HttpUrlEncoded.kt

```diff
@@ -31,23 +43,45 @@ fun String.parseUrlEncodedParameters(defaultEncoding: Charset = Charsets.UTF_8,
 /**
  * Encode form parameters from a list of pairs
  */
-fun List<Pair<String, String?>>.formUrlEncode(): String = StringBuilder().apply {
-    formUrlEncodeTo(this)
-}.toString()
+@Deprecated(
+    "Use formUrlEncode(option)",
+    ReplaceWith("formUrlEncode(UrlEncodingOption.DEFAULT)", "io.ktor.http.UrlEncodingOption"),
+    DeprecationLevel.HIDDEN
+)
+fun List<Pair<String, String?>>.formUrlEncode(): String = formUrlEncode(UrlEncodingOption.DEFAULT)
```

Comment:
Few notes about `Deprecated` here and other places:
1. Change message here to "Binary compatibility"
2. Drop ReplaceWith, it will never used